# These sequences of ArmStates make up the trajectory.
ArmState[] rArm
ArmState[] lArm
duration[] timing

# These record the dominant reference frames for the right and left
# arms.
uint8 rRefFrame
uint8 lRefFrame
Landmark rRefFrameLandmark
Landmark lRefFrameLandmark
